app/controllers/ncr/work_orders_controller.rb

```diff
@@ -15,7 +15,7 @@ def create
 
     def edit
       if self.proposal.approved?
-        flash[:warning] = "You are about to modify a fully approved request. Changes will be logged and sent to approvers, and this request may require re-approval, depending on the change."
+        flash.now[:warning] = "You are about to modify a fully approved request. Changes will be logged and sent to approvers, and this request may require re-approval, depending on the change."
```

Comment:
there are 2 other stories cc'd in the tracker story: 

https://www.pivotaltracker.com/n/projects/1149728/stories/98398508
https://www.pivotaltracker.com/n/projects/1149728/stories/97035674

since this PR is pretty small, do you think it would make sense to address all while we are here? Also, persistence of other flash messages hasn't been reported per se, but it probably makes sense to not have persisting flash messages everywhere, right?